docker/run_jaxbtck.sh

```diff
@@ -51,15 +51,15 @@ export  JAVA_TOOL_OPTIONS="-Dfile.encoding=UTF8"
 
 wget $WGET_PROPS $GF_BUNDLE_URL -O latest-glassfish.zip
 unzip -o latest-glassfish.zip
-chmod -R 777 glassfish5
+chmod -R 777 glassfish6
 
 mkdir -p JAXB_REPORT/JAXB-TCK
 
 cd jaxb-tck-build/unzip/JAXB-TCK-2.3/lib
 
-$JAVA_HOME/bin/java -jar javatest.jar -batch -testsuite $WORKSPACE/jaxb-tck-build/unzip/JAXB-TCK-2.3 -open javasoft-multiJVM.jti -workdir -create $WORKSPACE/batch-multiJVM/work -set jck.env.jaxb.xsd_compiler.skipValidationOptional Yes -set jck.env.jaxb.xsd_compiler.testCompile.xjcCmd "/bin/sh $WORKSPACE/jaxb-tck-build/unzip/JAXB-TCK-2.3/linux/bin/xjc.sh" -set jck.env.jaxb.schemagen.run.jxcCmd "/bin/sh $WORKSPACE/jaxb-tck-build/unzip/JAXB-TCK-2.3/linux/bin/schemagen.sh" -set jck.env.jaxb.testExecute.otherEnvVars "JAVA_HOME=/opt/jdk1.8.0_191 JAXB_HOME=$WORKSPACE/jaxb-ri JAXB_ENDORSED=$WORKSPACE/jaxb-ri/mod/jakarta.xml.bind-api.jar" -set jck.env.jaxb.testExecute.otherOpts "-Djava.endorsed.dirs=$WORKSPACE/glassfish5/glassfish/modules/endorsed/" -set jck.env.jaxb.classes.jaxbClasses "$JAVA_HOME/lib/tools.jar $WORKSPACE/jaxb-ri/mod/jakarta.xml.bind-api.jar $WORKSPACE/jaxb-ri/mod/jaxb-impl.jar $WORKSPACE/jaxb-ri/mod/jaxb-jxc.jar $WORKSPACE/jaxb-ri/mod/jakarta.activation.jar $WORKSPACE/glassfish5/glassfish/modules/jersey-media-jaxb.jar" -set jck.env.jaxb.classes.needJaxbClasses Yes -runtests
+$JAVA_HOME/bin/java -jar javatest.jar -batch -testsuite $WORKSPACE/jaxb-tck-build/unzip/JAXB-TCK-2.3 -open javasoft-multiJVM.jti -workdir -create $WORKSPACE/batch-multiJVM/work -set jck.env.jaxb.xsd_compiler.skipValidationOptional Yes -set jck.env.jaxb.xsd_compiler.testCompile.xjcCmd "/bin/sh $WORKSPACE/jaxb-tck-build/unzip/JAXB-TCK-2.3/linux/bin/xjc.sh" -set jck.env.jaxb.schemagen.run.jxcCmd "/bin/sh $WORKSPACE/jaxb-tck-build/unzip/JAXB-TCK-2.3/linux/bin/schemagen.sh" -set jck.env.jaxb.testExecute.otherEnvVars "JAVA_HOME=/opt/jdk1.8.0_191 JAXB_HOME=$WORKSPACE/glassfish6/glassfish JAXB_ENDORSED=$WORKSPACE/glassfish6/glassfish/modules/jakarta.xml.bind-api.jar" -set jck.env.jaxb.classes.jaxbClasses "$WORKSPACE/glassfish6/glassfish/modules/jakarta.xml.bind-api.jar $WORKSPACE/glassfish6/glassfish/modules/jaxb-osgi.jar $WORKSPACE/glassfish6/glassfish/modules/jersey-media-jaxb.jar $WORKSPACE/glassfish6/glassfish/modules/jakarta.activation.jar $JAVA_HOME/lib/tools.jar" -set jck.env.jaxb.classes.needJaxbClasses Yes -runtests
```

Comment:
what is this reason behind this change? Is it the switch from testing standalone JAXB impl to the implementation included in GF? Also note that original command was probably missing "jaxb-core.jar" on the classpath

Reply:
the original run script can be found @ https://github.com/eclipse-ee4j/jaxb-tck/commit/f0d0120ddae3eb276bf29f74dfa4699fb92c952c. during that time glassfish6 build was not available, for testing we started to use RI jars. Now glassfish build is available, hence reverted to glassfish 6 jars from RI jars.